Superbuild fails in Windows Debug

I created a fresh clone of Drake and attempted a Superbuild in Visual Studio Debug (Win64), which I have done successfully in the recent past. This time it failed, because lcm was apparently looking for `python27_d.lib` which would be a debug version of python if such a thing existed (is it supposed to?). I speculate that this is due to some otherwise-innocent recent CMakeLists change that caused CMake to add its standard debug suffix here where it wasn't before. Here is the error:
```
10>  Link:
10>    C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\x86_amd64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\Users\Sherm\Documents\GitHub\sherm2\drake-distro-build_d\externals\lcm\lib\python2.7\site-packages\lcm\Debug\_lcm.pyd" /INCREMENTAL /NOLOGO kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib ..\lcm\Debug\lcm.lib C:\Python27\libs\python27.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /DEBUG /PDB:"C:/Users/Sherm/Documents/GitHub/sherm2/drake-distro-build_d/externals/lcm/lib/python2.7/site-packages/lcm/Debug/_lcm.pdb" /SUBSYSTEM:CONSOLE /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:/Users/Sherm/Documents/GitHub/sherm2/drake-distro-build_d/externals/lcm/lcm-python/Debug/lcm-python.lib" /MACHINE:X64  /machine:x64 /DLL "lcm-python.dir\Debug\module.obj"
10>    "lcm-python.dir\Debug\pyeventlog.obj"
10>    "lcm-python.dir\Debug\pylcm.obj"
10>    "lcm-python.dir\Debug\pylcm_subscription.obj"
10>LINK : fatal error LNK1104: cannot open file 'python27_d.lib' [C:\Users\Sherm\Documents\GitHub\sherm2\drake-distro-build_d\externals\lcm\lcm-python\lcm-python.vcxproj]
10>  Done Building Project "C:\Users\Sherm\Documents\GitHub\sherm2\drake-distro-build_d\externals\lcm\lcm-python\lcm-python.vcxproj" (default targets) -- FAILED.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -211,6 +211,17 @@ set(lcm_ADDITIONAL_CMAKE_CONFIGURE_ARGS -DBUILD_SHARED_LIBS=ON)
 if(WITH_GTK)
   set(lcm_ADDITIONAL_CMAKE_CONFIGURE_ARGS ${lcm_ADDITIONAL_CMAKE_CONFIGURE_ARGS} -DCMAKE_PREFIX_PATH=${CMAKE_CURRENT_SOURCE_DIR}/externals/gtk/gtk3)
 endif()
+if(WIN32)
+  # On Windows we always need a Release LCM for compatibility with Python libraries.
+  if(CMAKE_CONFIGURATION_TYPES)
+    # This is a multi-config generator, so we need to specify the configuration at build time.
+    set(lcm_BUILD_COMMAND ${CMAKE_COMMAND} --build . --config Release)
+    set(lcm_INSTALL_COMMAND ${CMAKE_COMMAND} --build . --config Release --target install)
+  else()
+    # This is a single-config generator, so we need to specify the configuration to CMake.
+    list(APPEND lcm_ADDITIONAL_CMAKE_CONFIGURE_ARGS -DCMAKE_BUILD_TYPE:STRING=Release)
+  endif()
+endif()
 set(libbot_IS_CMAKE_POD TRUE)
 set(libbot_IS_PUBLIC TRUE)
 set(bot_core_lcmtypes_IS_CMAKE_POD TRUE)
@@ -372,12 +383,21 @@ foreach(proj ${EXTERNAL_PROJECTS})
       if(CMAKE_VERBOSE_MAKEFILE)
         set(PODS_VERBOSE_MAKEFILE "-DCMAKE_VERBOSE_MAKEFILE=ON")
       endif()
+      foreach(c BUILD INSTALL)
+        if(DEFINED ${proj}_${c}_COMMAND)
+          set(maybe_${c}_COMMAND ${c}_COMMAND ${${proj}_${c}_COMMAND})
+        else()
+          set(maybe_${c}_COMMAND "")
+        endif()
+      endforeach()
       ExternalProject_Add(${proj}
         SOURCE_DIR ${${proj}_SOURCE_DIR}
         BINARY_DIR ${${proj}_BINARY_DIR}
         DOWNLOAD_COMMAND ${${proj}_DOWNLOAD_COMMAND}
         DOWNLOAD_DIR ${CMAKE_CURRENT_SOURCE_DIR}
         UPDATE_COMMAND ${${proj}_UPDATE_COMMAND}
+        ${maybe_BUILD_COMMAND}
+        ${maybe_INSTALL_COMMAND}
         BUILD_ALWAYS 1
         INDEPENDENT_STEP_TARGETS update
         DEPENDS ${deps}

